sample/Kmd.Logic.CitizenDocuments.Client.Sample/Program.cs

```diff
@@ -75,6 +74,7 @@ private static async Task Run(AppConfiguration configuration)
                     Log.Error("There is error occured in upload");
                     return;
                 }
+
                 Log.Information($"Document uploaded successfully and details are :-  DocumentId : {uploadDocument.DocumentId} ; DocumentType : {uploadDocument.DocumentType} ; FileAccessPageUrl : {uploadDocument.FileAccessPageUrl} ");
```

Comment:
Avoid using string interpolation with Serilog.

Reply:
Ok. Removed string interpolation at all the places where logging events with serilog.